apps/application-system/api/src/app/modules/application/files/file.service.ts

```diff
@@ -71,14 +73,18 @@ export class FileService {
   ): Promise<SigningServiceResponse> {
     this.validateApplicationType(application.typeId)
     const { answers, externalData, id, state } = application as CRCApplication
-    const { nationalRegistry, parentNationalRegistry } = externalData
+    const { nationalRegistry } = externalData
     const isParentA = state === 'draft'
```

Comment:
Not in this PR but I always think when I see this that we should change this check.

I am thinking if we should send in the authenticated user national id to the function and check if that is the same as the `application.applicant` national id. Any thoughts about this?

Reply:
Yeah, that is definitely nicer than this check 💯